.env

```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=mongodb://localhost/anbu-cfh
```

Comment:
This file should be added to your .gitignore you don't want to share some sensitive information in the repo.